core/encoding.rbs

```diff
@@ -435,16 +435,6 @@ class Encoding < Object
   #
   def names: () -> ::Array[String]
 
-  # <!--
-  #   rdoc-file=encoding.c
-  #   - enc.replicate(name) -> encoding
-  # -->
-  # Returns a replicated encoding of *enc* whose name is *name*. The new encoding
-  # should have the same byte structure of *enc*. If *name* is used by another
-  # encoding, raise ArgumentError.
-  #
-  def replicate: (String name) -> Encoding
```

Comment:
Removing should be fine according to https://github.com/ruby/rbs/blob/master/docs/CONTRIBUTING.md#how-can-we-handle-incompatibilities-of-core-apis-and-standard-libraries-between-rubies